# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/cabal-install/hotfix-cabal.patch
# Copyright (C) 2026 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- a/cabal-install.cabal
+++ b/cabal-install.cabal
@@ -253,7 +253,7 @@ library
       , hooks-exe  >= 3.18     && < 3.19
       , HTTP       >= 4000.1.5 && < 4000.6
       , mtl        >= 2.0      && < 2.4
-      , network-uri >= 2.6.2.0 && < 2.7
+      , network-uri
       , pretty     >= 1.1      && < 1.2
       , process    >= 1.2.3.0  && < 1.6.24 || == 1.6.26.0 || >= 1.6.26.2 && < 1.7
       , random     >= 1.2      && < 1.4
